BuildScripts/CopyProjDirectory.bat

```diff
@@ -0,0 +1,6 @@
+rem File "d:\checkouts\D-FAST\D-FAST_Morphological_Impact\dfastmi.dist\pyproj\datadir.py", line 109, in get_data_dir
```

Comment:
Lets clean up this comment so it does not have a developer's path in the comment.

Reply:
Indeed, this was a verbatim copy of a run-time error message that got and made it easy to compare with previously encountered messages, but now that we're building in different environments this should at least refer to relative paths.